torch/autograd/function.py

```diff
@@ -372,3 +381,71 @@ def forward_extended(self, *input):
 
     def backward_extended(self, *grad_output):
         raise NotImplementedError
+
+
+def symbolic_override(symbolic_fn):
```

Comment:
I don't think this belongs to autograd. Can you move this to where ONNX code lives?